package.json

```diff
@@ -88,15 +86,10 @@
     "build:fat": "yarn run compile-config-schema && tsc -p tsconfig.build.json",
     "build:watch": "yarn run compile-config-schema && tsc -p tsconfig.build.json --watch",
     "build": "yarn compile-config-schema && esbuild src/index.ts --bundle --platform=node --target=node10.8 --outfile=dist/craft --minify",
-    "precli": "run-s build",
+    "precli": "build",
     "cli": "node dist/craft",
     "clean": "rimraf dist coverage",
-    "lint": "run-s lint:prettier lint:eslint",
-    "lint:prettier": "prettier-check 'src/**/*.ts'",
-    "lint:eslint": "eslint src  --ext .ts --format stylish",
-    "fix": "run-s fix:tslint fix:prettier",
-    "fix:prettier": "prettier --write 'src/**/*.ts'",
-    "fix:eslint": "eslint src --ext .ts --fix --format stylish",
+    "lint": "prettier --write .",
```

Comment:
There is common terminology with `sentry-javascript`, and even with Makefiles in other repos where:
- `lint` means read-only liniting
- `fix` means writing back changes

I'd prefer to keep this consistent. I know this isn't documented yet, will add this to develop docs when I get the chance.

Reply:
I don't find read-only linting useful as if something can be automatically fixed, then it _should be_ automatically fixed. What am I missing here?

(I get the unspoken convention, just challenging it before I adopt)

Reply:
There are cases where automatic fixes can't or shouldn't be applied. In most languages, we also lint for conventions, code structure, etc.

Reply:
Unless you present me specific cases, I'll go ahead and keep auto-fix in-place until we find a need to switch to read-only mode.

Reply:
The specific case is 2/3 of all Rust lints that are not auto-fixable. 

Let me turn this around: About two years ago we set the convention of `lint` and `fix` in every repo. Unless there is a hard reason why we **must** diverge from that convention in craft, we should default to keeping it consistent, please.

Reply:
>The specific case is 2/3 of all Rust lints that are not auto-fixable. (Edit: I'm not to familiar with latest advancements in JS, but there's gotta be some more advanced linting than just reformatting)

This is JS though and we are just formatting code here with linting :)

>Let me turn this around: About two years ago we set the convention of lint and fix in every repo. Unless there is a hard reason why we must diverge from that convention in craft, we should default to keeping it consistent, please.

This is just a way to block progress for the sake of status quo. If you have cases for JS, I'll see how they affect this, otherwise please do not block simplifications with unclear or unmaterialized concerns.

Reply:
> I'll go ahead and keep auto-fix

Can you spell it `fix` instead of `lint`? I think that would address @jan-auer's concern. For comparison, [`sentry-cli` has `fix` scripts and no `lint` scripts](https://github.com/getsentry/sentry-cli/blob/d309c5a607a310eebd23c1a2e49624e6ea192e21/package.json#L26-L36).

Reply:
Done.

Reply:
Does that help, @jan-auer?

Reply:
That works :)